docs/notifications-migration.rst

```diff
@@ -0,0 +1,73 @@
+.. _notifications-migration-guide:
+
+Notifications to Alerts Migration
+=================================
+
+Use this guide to update from using ```get_notifications()```, which leverages the
+```/integrationServices/v3/notification``` API to using Alerts in SDK v1.5.0 or higher with Alerts v7 API.
+
+.. note::
+    The /integrationServices/v3/notificationf API is deprecated, and deactivation is planned for 31 October 2024.
+
+    The Access Level Type SIEM used to access the Notifications API is also deprecated, and deactivation is planned for 31 January 2025.
```

Comment:
I'm confused with the 31 January 2025 date as 31 October 2024 seems to be the actual date

Reply:
Clarified wording.